src/Tests/Indices/Monitoring/IndicesStats/IndicesStatsWithShardsInformationApiTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Linq;
+using Elasticsearch.Net;
+using FluentAssertions;
+using Nest;
+using Tests.Framework;
+using Tests.Framework.Integration;
+using Tests.Framework.ManagedElasticsearch.Clusters;
+
+namespace Tests.Indices.Monitoring.IndicesStats
+{
+	public class IndicesStatsWithShardsInformationApiTests : ApiIntegrationTestBase<WritableCluster, IIndicesStatsResponse,
+		IIndicesStatsRequest, IndicesStatsDescriptor, IndicesStatsRequest>
+	{
+		public IndicesStatsWithShardsInformationApiTests(WritableCluster cluster, EndpointUsage usage) : base(cluster, usage)
+		{
+		}
+
+		private class NoopClass
+		{
+		}
+
+		protected override void IntegrationSetup(IElasticClient client, CallUniqueValues values)
+		{
+			if (this.Client.IndexExists(typeof(NoopClass)).Exists)
+			{
+				return;
```

Comment:
pedantic but lose the brackets for early exits (single line them).

Reply:
Done.